src/Client.php

```diff
@@ -707,7 +733,7 @@ private function request($method, $url, array $options = [])
             }
 
             throw new ClientException(
-                isset($body['error']) ? $body['error'] : $e->getMessage(),
+                $this->composeErrorMessage($e, $body),
```

Comment:
Proč se ta API error message překládá takhle do stringu? Klidně by mohla mít taky extra parametr...

Reply:
Jakože PHP Client může vyhodit message `Invalid request, check "details" exception property for more info`. Dokonce by mi to přišlo i užitečnější...